website/docs/addon_maya_ornatrix_admin.md

```diff
@@ -0,0 +1,65 @@
+---
+id: addon_maya_ornatrix_admin
+title: Ornatrix for Maya
+sidebar_label: Ornatrix
+description: AYON Maya Ornatrix Admin documentation
+toc_max_heading_level: 5
+---
+
+import ReactMarkdown from "react-markdown";
+import versions from '@site/docs/assets/json/Ayon_addons_version.json'
+
+<ReactMarkdown>
+{versions.Maya_Badge}
+</ReactMarkdown>
+
+
+## Maya Addon Setting for Ornatrix Maya
+
+### Creators
+
+At present, we support two Orantrix product types. You can let artists create these products by enabling them in the Maya create settings.
+
+- `ayon+settings://maya/create/CreateOxRig`
+- `ayon+settings://maya/create/CreateOxCache`
+
+![](assets/maya/admin/ornatrix_creators.png)
+
+### Validators
+
+Currently, we provide content validation for the two Orantrix product types.
```

Comment:
```suggestion
Currently, we provide content validation for the two Ornatrix product types.
```

Reply:
fixed in https://github.com/ynput/ayon-documentation/pull/237/commits/7a84098e9fe5327cfc62278b0b720399bf206088